align/primitive/main.py

```diff
@@ -174,7 +174,7 @@ def generate_primitive(block_name, primitive, height=28, x_cells=1, y_cells=1, p
         uc, _ = generate_generic(pdkdir, parameters, netlistdir=netlistdir)
     elif 'ring' in primitive:
         uc, _ = generate_Ring(pdkdir, block_name, x_cells, y_cells)
-    elif 'MOS' == primitive.generator['name']:
+    elif 'MOS' == primitive.generator['name'].upper():
```

Comment:
@stevenmburns the test failure you highlighted in this PR just needs this line update. Rest updates are coming from another PR.